### DISCUSSION TITLE
Are there any plans on optimizing retries?

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#21251] [Q&A] [answered]
In stable on a test map it takes a little under 3 seconds to go from hitting the retry button and arriving to the first hit object (also the map loads instantly instead of having a waiting screen every time.) while in lazer it takes around 5 seconds (due to the ~2 second loading screen.)
Additionally, having a loading screen every single time you retry is a little jarring. Im sure the game doesn't need to load the map from the disk every time you retry, so either the waiting time is hard coded (annoying game design), or the game tries loading the map each time (inefficent), or maybe something else is causing this, but either way, it should be changed to make the process a lot less noticable.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/9039
